src/constants/ReadTime/ReadTime.jsx

```diff
@@ -0,0 +1,9 @@
+
+
+function berekenLeestijd(tekst) {
```

Comment:
Functienaam in het Engels voor herbruikbaarheid. Verder goed gemaakt!! Ik kreeg het niet voor elkaar

Ik zie ook dat deze in de map constants staat maar het is een helper functie :)